docs/developer-docs/integrations/sns/managing/managing-nervous-system-parameters.md

```diff
@@ -159,3 +159,139 @@ quill send message.json
 ```
 
 See example [proposal of an active SNS](https://dashboard.internetcomputer.org/sns/7jkta-eyaaa-aaaaq-aaarq-cai/proposal/6).
+
+## The parameters
+
+### `default_followees`
+
+The set of default followees that every newly created neuron will follow per function. This is specified as a mapping of proposal functions to followees. If unset, neurons will have no followees by default. The set of followees for each function can be at most of size `max_followees_per_function` parameter.
+
+### `max_dissolve_delay_seconds`
+  
+The maximum dissolve delay that a neuron can have. That is, the maximum
+that a neuron's dissolve delay can be increased to. The maximum is also enforced
+when saturating the dissolve delay bonus in the voting power computation.
+
+### `max_dissolve_delay_bonus_percentage`
+
+E.g. if a large dissolve delay can double the voting power of a neuron,
+then this field would have a value of 100, indicating a maximum of
+100% additional voting power.
+
+For no bonus, this should be set to 0.
+
+To achieve functionality equivalent to NNS, this should be set to 100.
+
+### `max_followees_per_function`
+
+The maximum number of followees each neuron can establish for each nervous system function.
+
+This number can be at most as large as the defined ceiling
+`MAX_FOLLOWEES_PER_FUNCTION_CEILING` (15 at time of writing).
+
+### `neuron_claimer_permissions`
+
+The default set of neuron permissions granted to the principal claiming a neuron.
+
+### `neuron_minimum_stake_e8s`
+
+The minimum number of e8s (10E-8 of a token) that can be staked in a neuron.
+
+To ensure that staking and disbursing of the neuron work, the chosen value
+must be larger than the `transaction_fee_e8s`.
+
+### `max_neuron_age_for_age_bonus`
+
+The age of a neuron that saturates the age bonus for the voting power computation.
```

Comment:
```suggestion
The maximum age of a neuron, defined in seconds, that saturates the age bonus for the voting power computation.
```

@daniel-wong-dfinity-org do you recall if this is indeed in seconds? Since it is not in the name I wasn't 100% sure, but assume so as the age itself is also in age IIRC